assets/questions/aad.js

```diff
@@ -0,0 +1,309 @@
+        "cpctfa",
+        "cpbjeq",
+        "cpbjec",
+        "cpbjz",
+        "cpbhx",
+        "cpbhc",
+        "khjzs",
+        "khjzu",
+        "khjis",
+        "khjiv",
+        "khfax",
+        "khfay",
+        "khftg",
+        "khftq",
+        "kmima",
+        "kmimd",
+        "kmiej",
+        "kmies",
+        "kmrbm",
+        "kmrbr",
+        "kmrus",
+        "kmrut"
+      ],
+      allowedOutputs: ['ljhmezirxygadoqckfsvntwbup']
+    }],
+    buggyOutputTests: [],
+    performanceTests: []
+  }, {
+    instructions: [
+      [
+        'Next, double-check your code to make sure it handles unexpected cases',
+        ', such as there\'s no solution to the test case or there\'re more ',
+        'than one answers.'
+      ].join(''),
+      [
+        'For the test case that has no solution, just return "". For the test ',
+        'case that has multi-answers, return the answer has the smallest ',
+        'lexicographic order in English alphabet.'
+      ].join('')
+    ],
+    prerequisiteSkills: ['Topology Sorting', 'String Manipulation'],
+    acquiredSkills: ['String Manipulation'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'find_dict',
+    correctnessTests: [{
+      input: [
+        "a",
+        "b",
+        "a"
+      ],
+      allowedOutputs: ['']
+    }, {
+      input: [
+        "c",
+        "a",
+        "b",
+        "a"
+      ],
+      allowedOutputs: ['']
+    }, {
+      input: [
+        "cab",
+        "bc"
+      ],
+      allowedOutputs: ['acb']
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.wrongInNoSolution',
+      messages: [
+        [
+          'You may try this case : ["c", "a", "b", "a"]'
+        ].join('')
+      ]
+    }],
+    performanceTests: []
+  }, {
+    instructions: [
+      [
+        'Last, Let\'s fix the solution to adapt lowercase and uppercase. We ',
+        'take the lowercase and uppercase of the same letter with equal ',
+        'lexicographic order. And then return the answer in lowercase'
+      ].join('')
+    ],
+    prerequisiteSkills: ['Topology Sorting', 'String Manipulation'],
+    acquiredSkills: ['String Manipulation'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'find_dict',
+    correctnessTests: [{
+      input: [
+        "ab",
+        "Aba",
+        "b"
+      ],
+      allowedOutputs: ['ab']
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.wrongInUppercase',
+      messages: [
+        [
+          'You may try this case : ["a", "Ab", "b"]'
+        ].join(''),
+        [
+          'The answer to the test case ["a", "Ab", "b"] should be "ab".'
+        ].join('')
+      ]
+    }],
+    performanceTests: []
+  }],
+  styleTests: [{
+    evaluationFunctionName: 'allowOnlyOneFunction',
+    expectedOutput: true,
+    message: [
+      'You should only be writing code in an encode function. While ',
```

Comment:
Please don't leave text in your code that corresponds to other questions. Update this to be consistent with your question.